index.html

```diff
@@ -259,17 +259,17 @@ <h1>About Me</h1>
             
                 
                     <textarea class="text" name="msg" maxlength="500" placeholder="Enter text here" required></textarea>
-                
+                </form>
                 <div class="button-cont">
-                    <button type="submit" class="potfolio-but">Get it Touch</button>
+                    <button type="submit" class="potfolio-but">Get in Touch</button>
```

Comment:
- [ ] This submit button should be within the form element. Kindly move this submit button into the form, just before the `</form>` closing tag

Reply:
done